packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -0,0 +1,159 @@
+Terra.describeViewports('Dropdown Button', ['medium'], () => {
+  describe('Default', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-dropdown-button/dropdown-button/default-dropdown-button');
+      // avoid hover styles
+      browser.moveToObject('#root', 100, 100);
+    });
+
+    Terra.it.validatesElement();
+
+    it('validates focus styling', () => {
+      browser.keys('Tab');
+      Terra.validates.element('focus');
+      // tab away from the button so focus is not in later screenshots
+      browser.keys('Tab');
+    });
+
+    it('validates hover styling', () => {
+      browser.moveToObject('[class*=dropdown-button-type]');
+      Terra.validates.element('hover');
+    });
+
+    it('validates the open dropdown', () => {
+      browser.click('[class*=dropdown-button-type]');
+    });
+
+    Terra.it.validatesElement('dropdown open', { selector: '[class*=dropdown-list]' });
+    Terra.it.matchesScreenshot('dropdown button with dropdown open');
+  });
+
+  describe('Disabled', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-dropdown-button/dropdown-button/disabled-dropdown-button');
+    });
+
+    Terra.it.validatesElement();
```

Comment:
Because this is composed of multiple components, it'd probably be good to add a test to try to tab to the component and verify it did not receive focus